scripts/generate_dump

```diff
@@ -1791,6 +1812,8 @@ main() {
 
     if [[ "$device_type" != "SpineRouter" ]]; then
         save_saidump
+    else
+        save_saidump_by_redis_save_cmd
```

Comment:
i do not feel we should check device_type, and doing different saidump based on the device type. Can you check the asic db size and if it is larger than xxx entries, then we do the new way?

Reply:
@lguohan, please help to review my updates. The variable ROUTE_TAB_LIMIT_DIRECT_ITERATION with default value 6400 needs to be tuned and provided by my colleagues.